ironfish/src/workerPool/tasks/workerMessage.ts

```diff
@@ -20,7 +20,7 @@ export enum WorkerMessageType {
   BuildTransaction = 9,
 }
 
-export abstract class WorkerMessage implements Serializable {
+export abstract class WorkerMessage {
```

Comment:
why did you remove `Serializable`?

Reply:
Leftover from a previous change; reverting. Thanks for catching this.